src/SayMore/UI/ExportMediaDlg.cs

```diff
@@ -0,0 +1,260 @@
+using System;
+using System.ComponentModel;
+using System.Drawing;
+using System.Windows.Forms;
+using L10NSharp;
+using L10NSharp.UI;
+using SIL.Reporting;
+using SIL.Windows.Forms;
+using SIL.Windows.Forms.PortableSettingsProvider;
+using SayMore.Media.FFmpeg;
+using SayMore.Properties;
+
+namespace SayMore.UI
+{
+	public partial class ExportMediaDlg : Form
```

Comment:
If it's not really feasible to have a single dialog that can be parameterized to provide the two different desired behaviors, we at least need a comment near the top of both versions pointing out that they have a lot of similar code that generally should be kept in sync so that improvements in the one are also made in the other.

Reply:
Combined into one.